docs/mycoind/iovcore.rst

```diff
@@ -40,7 +40,11 @@ Using Iov-Core
 
 Please refer to the offical `iov-core documentation <https://github.com/iov-one/iov-core/blob/master/packages/iov-core/README.md>`__
 Note that you can use the ``BnsConnection`` to connect to a ``mycoind`` blockchain, as long as you restrict it to just sending tokens
-and querying balances and nonces (it is a subset of bnsd).
+and querying balances and nonces (it is a subset of bnsd). You may also find
```

Comment:
I think it's good to somehow format the names like `bnsd` (like I did here). This might help newcomers to not get confused by words they don't know. I expect that if they see it formatted differently, they will figure out that this is not a typo or a word they don't know but a name. It would be awesome to be able to generate an `<abbr>` tag.